card/spec/lib/card/query/card_query/special_attributes_spec.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "../query_spec_helper"
+RSpec.describe Card::Query::CardQuery::SpecialAttributes do
+  include QuerySpecHelper
+
+  # TODO: add specs for: #name_match, #complete, #junction_complete
+
+  describe "match" do
+    it "reachs content and name via shortcut" do
+      @query = { match: "two" }
+      is_expected.to eq(cards_matching_two)
+    end
+
+    it "gets only content when content is explicit" do
+      @query = { content: [:match, "two"] }
+      is_expected.to eq(["Joe User"])
+    end
+
+    it "gets only name when name is explicit" do
+      @query = { name: [:match, "two"] }
+      is_expected.to eq(["One+Two", "One+Two+Three", "Two"])
+    end
+
+    context "with keyword" do
+      it "escapes nonword characters" do
+        @query = { match: "two :(!" }
+        is_expected.to eq(cards_matching_two)
+      end
+
+      it "it can handle *" do
+        @query = { match: "*all" }
+        is_expected.to include("*all")
+      end
+    end
+  end
+
+  describe "found_by" do
+    before do
+      Card::Auth.as_bot
```

Comment:
I think doing it like this makes it persistent through all tests executed afterwards
Better use the `as_bot: true` option.

Reply:
for now I just used the as_bot block.  That way the queries are run with standard permissions.